task-09/README.md

```diff
@@ -0,0 +1,57 @@
+[previous task](../task-08)
+
+# Task 9: RIOT and RPL
+
+This task will demonstrate basic routing functionalities in RIOT provided by the [RPL](https://tools.ietf.org/html/rfc6550) (IPv6 Routing Protocol for Low-Power and Lossy Networks) routing protocol and multihop communication.
+
+It uses an adapted version of the default [`gnrc_networking`](https://github.com/RIOT-OS/RIOT/tree/master/examples/gnrc_networking) example in the RIOT repository.
+
+## Prerequisites
+
+* Compile the application and run on your `BOARD`
+* Build groups of at least three persons
+* Each group should change to an individual RF channel. It can be changed by e.g.
+
+```
+ifconfig 7 set chan 11
```

Comment:
Please be more generic (use `<if_id>` instead of `7`). `7` could change in the future